lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/unordered/main.cpp

```diff
@@ -18,15 +18,19 @@ int main() {
   char buffer[sizeof(std::unordered_map<int, std::string>)] = {0};
   std::unordered_map<int, std::string> &corrupt_map = *(std::unordered_map<int, std::string> *)buffer;
 
-  std::unordered_map<int, std::string> map; // Set break point at this line.
+  // Make a typedef to ensure functionality when typedefs are used.
+  typedef std::unordered_map<int, std::string> UnorderedMap;
```

Comment:
I think we should re-add at least one test case to ensure we're testing with and without the typedef.